packages/manager/apps/dedicated/client/app/models/BillingService.class.js

```diff
@@ -82,7 +82,7 @@ export default class BillingService {
   }
 
   hasEngagement() {
-    return !isNull(this.engagedUpTo);
+    return !isNull(this.engagedUpTo) && moment().isBefore(this.engagedUpTo);
```

Comment:
not a big deal but i think it's safer to use /auth/time api call to get the current time, otherwise we rely on the client's browser date settings

Reply:
You are right, but it looks like a Manager-wide solution is needed for this :)

Also, this is not the scope of this issue

Reply:
@frenautvh indeed sounds like a better solution but as @FredericEspiau said we might need to think of a way to globally use /auth/time